scripts/evaluate_gaussian_features.py

```diff
@@ -0,0 +1,37 @@
+from multiprocessing.pool import Pool
```

Comment:
Probably not worth adding, this is just for internal purposes

Reply:
When is runned on multiple length of gaussian it have sens. Current state is not harmful imho.

Reply:
I still would prefer to remove it ;)